src/components/htmlvideoplayer/plugin.js

```diff
@@ -686,27 +711,30 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
             }).length > 0;
         }
 
-        function getSupportedAudioStreams() {
-            var profile = self._lastProfile;
+        /**
+         * @private
+         */
+        getSupportedAudioStreams() {
+            const profile = self._lastProfile;
 
-            return getMediaStreamAudioTracks(self._currentPlayOptions.mediaSource).filter(function (stream) {
-                return isAudioStreamSupported(stream, profile);
+            return getMediaStreamAudioTracks(self._currentPlayOptions.mediaSource).filter((stream) => {
+                return this.isAudioStreamSupported(stream, profile);
             });
         }
 
-        self.setAudioStreamIndex = function (index) {
+        setAudioStreamIndex(index) {
 
-            var streams = getSupportedAudioStreams();
+            const streams = this.getSupportedAudioStreams();
 
             if (streams.length < 2) {
                 // If there's only one supported stream then trust that the player will handle it on it's own
                 return;
             }
 
-            var audioIndex = -1;
-            var i;
-            var length;
-            var stream;
+            let audioIndex = -1;
+            let i;
+            let length;
+            let stream;
 
             for (i = 0, length = streams.length; i < length; i++) {
```

Comment:
Replace with `forEach()`

Reply:
Will do, but couldn't that also just be for of? Makes it a bit more readable